### ISSUE TITLE
Allow periods in @import under `clean-import-paths`

### ISSUE THREAD

**Issue by waldemarfm** [sasstools/sass-lint#553]
I like to use periods in my file names.

```scss
@import 'components.header`;
```

Using sass-lint 1.5.1, the `clean-import-paths` config option throws the warning `File extensions are not allowed` but I'm not including `.scss` in the import, just a period. Since you can't import anything other than SASS and maybe CSS files, maybe there should be better detection of this and throw the warning only if the import string ends in `.scss` and `.css`, otherwise allow periods.

**Comment by bgriffith** [comment]
I have a feeling this is already fixed on develop branch and will be released with 1.6.

**Comment by erikdesjardins** [comment]
Yep, looks like #536
